recipes/fasd/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "fasd" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/clvv/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 88efdfbbed8df408699a14fa6c567450bf86480f5ff3dde42d0b3e1dee731f65
+
+build:
+  number: 0
+  script: "make install"
```

Comment:
It looks like you should add `skip: true  # [win]` under the `build` section since this looks to be a unix-only utility.

Reply:
Rather this should be `skip: True  # [win]`

Reply:
> Rather this should be `skip: True # [win]`
thanks. Added build skip.

Reply:
Do you need to pass in the prefix here or anything?